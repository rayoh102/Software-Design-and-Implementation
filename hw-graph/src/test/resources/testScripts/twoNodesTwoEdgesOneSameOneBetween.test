# Tests adding two nodes, one edge on the same node and one edge between two nodes to graph

# Create a graph
CreateGraph twoNodesTwoEdgesOneSameOneBetween

# Add two nodes
AddNode twoNodesTwoEdgesOneSameOneBetween
AddNode twoNodesTwoEdgesOneSameOneBetween

# Add two edges
AddEdge twoNodesTwoEdgesOneSameOneBetween
AddEdge twoNodesTwoEdgesOneSameOneBetween

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
ListNodes ttwoNodesTwoEdgesOneSameOneBetween
ListChildren twoNodesTwoEdgesOneSameOneBetween a
ListChildren twoNodesTwoEdgesOneSameOneBetween b