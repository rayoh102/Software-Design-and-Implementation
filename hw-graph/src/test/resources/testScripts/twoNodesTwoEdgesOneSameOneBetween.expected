# Tests adding two nodes, one edge on the same node and one edge between two nodes to graph

# Create a graph
created graph twoNodesTwoEdgesOneSameOneBetween

# Add two nodes
added node a to twoNodesTwoEdgesOneSameOneBetween
added node b to twoNodesTwoEdgesOneSameOneBetween

# Add two edges
added edge AA from a to a in twoNodesTwoEdgesOneSameOneBetween
added edge AB from a to b in twoNodesTwoEdgesOneSameOneBetween

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
twoNodesTwoEdgesOneSameOneBetween contains: a b
the children of a in twoNodesTwoEdgesOneSameOneBetween are: a(AA) b(AB)
the children of b in twoNodesTwoEdgesOneSameOneBetween are: